modules/getting-started-web-console-login.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-web-console.adoc
+
+[id="getting-started-web-console-login_{context}"]
+= Logging into the web console
+
+You can log in to the {product-title} web console to access and manage your cluster.
+
+.Prerequisites
+
+* {product-title} web console login credentials.
+
+.Procedure
+
+. Log in to the {product-title} web console using your login credentials.
+
+You are redirected to the *Projects* page. This is the default view for the {product-title} web console is the *Administrator* perspective.
```

Comment:
the text is confusing here,
for non-admin users, the default view is `Developer` perspective, while for cluster admin users the default view is `Administrator` perspective.